mackup/application.py

```diff
@@ -80,18 +99,23 @@ def backup(self):
                         # Delete the file in Mackup
                         utils.delete(mackup_filepath)
                         # Copy the file
-                        utils.copy(filepath, mackup_filepath)
+                        utils.copy(home_filepath, mackup_filepath)
                         # Delete the file in the home
-                        utils.delete(filepath)
+                        utils.delete(home_filepath)
                         # Link the backuped file to its original place
-                        utils.link(mackup_filepath, filepath)
+                        utils.link(mackup_filepath, home_filepath)
                 else:
                     # Copy the file
-                    utils.copy(filepath, mackup_filepath)
+                    utils.copy(home_filepath, mackup_filepath)
                     # Delete the file in the home
-                    utils.delete(filepath)
+                    utils.delete(home_filepath)
                     # Link the backuped file to its original place
-                    utils.link(mackup_filepath, filepath)
+                    utils.link(mackup_filepath, home_filepath)
+            elif self.verbose:
+              print("Doing nothing\n  {}\n  {}".format(
+                  home_filepath, 
+                  "is already backed up to\n  " + mackup_filepath
```

Comment:
"is already backed up to does not exist"? Feel free to put a more complex conditions here to get a wording that's gonna be easier to understand.